api/capi/include/nnstreamer.h

```diff
@@ -128,6 +128,7 @@ typedef enum {
   ML_NNFW_TYPE_VIVANTE = 7,           /**< VeriSilicon's Vivante. (Since 6.0) */
   ML_NNFW_TYPE_EDGE_TPU = 8,          /**< Google Coral Edge TPU (USB). (Since 6.0) */
   ML_NNFW_TYPE_ARMNN = 9,             /**< Arm Neural Network framework (support for caffe and tensorflow-lite). (Since 6.0) */
+  ML_NNFW_TYPE_SNPE = 10,             /**< Qualcomm SNPE (Snapdgragon Neural Processing Engine (.dlc). (Since 6.0) */
```

Comment:
@jaeyun-jung , @again4you : Before staring the next ACR, we will need to clean up the doxygen entries. (especially with "Since Tizen" syntax)